conf/sample.ini

```diff
@@ -1308,7 +1309,7 @@
 ; enabled = false
 ; root_ca_cert =
 ; client_key =
-; client_cert = 
+; client_cert =
 ; server_name =
 # The address of the socks5 proxy datasources should connect to
-; proxy_address =
\ No newline at end of file
+; proxy_address =
```

Comment:
I'm not sure is it a blank line in the end of the file? Original one does not have it. Could you double check? Sometimes editor adds new line on save.